packages/lab/src/form-field-next/FormFieldNext.css

```diff
@@ -0,0 +1,62 @@
+.saltFormFieldNext {
+  border: 0;
+  display: inline-grid;
+  font-size: var(--saltFormFieldNext-fontSize, var(--salt-text-fontSize));
+  min-width: 0;
+  position: relative;
+  padding: 0;
+  column-gap: var(--salt-size-adornmentGap);
+  row-gap: var(--salt-size-adornmentGap);
+  vertical-align: top;
+  width: var(--saltFormFieldNext-width, 100%);
+}
+
+.saltFormFieldNext-labelLeft {
+  align-self: center;
+  grid-template-columns: auto 1fr;
+}
+
+.saltFormFieldNext-labelTop {
```

Comment:
do we need this class?

Reply:
I'm going to keep empty for now as the grid css might change and want to remind us this is how we're doing it